packages/core/src/components/modal/modal.scss

```diff
@@ -43,6 +44,7 @@
   &__header {
     color: var(--color-neutral-light-1);
     padding: var(--spacing-large);
+    padding-right: var(--spacing-xxxxlarge);
```

Comment:
question: is it because icon to close?

Reply:
Yes, it ends up on top of the title in certain resolutions when the title is very large. lol

Before

https://github.com/user-attachments/assets/ec3df326-11c6-44a2-8d1b-95bf657a3399

After

https://github.com/user-attachments/assets/c8b40714-7fd8-4979-a876-5d7a9777c5ff

Reply:
@gabrielduete ahhh is better to add some logic here creating a variable to provide size of icon area and use it here

Reply:
Nicee!

Made: https://github.com/juntossomosmais/atomium/pull/730/commits/d4d927d0cb369c0d02ff233a14dddaa697f1abd9